### PR TITLE
[Playwright] LPD-26753 The URL changes when a category is modified

### PR DESCRIPTION
**Pull request by boton**
# What is this trying to solve?

https://liferay.atlassian.net/browse/LPD-26753

We are adding a playwright test to check blog entry friendlyURL redirection using categories when categories are renamed

# How am I fixing it?

We are adding a functional test.

# How can you verify that it works?

1. Go to `/modules/test/playwright`
3. Run `npm install` (setup)
4. Run `npm run playwright test -- -g "LPD-26753"` to only run the test

# Avoid flakiness by repeating 10 times

```sh
npm run playwright test -- -g "LPD-26753" --reporter list --repeat-each 10
```
image

# Screenrecord test execution
[blogEntry-LPD-26753-The-URL-changes-when-a-category-is-modified-blogs-web.webm](https://github.com/liferay-content-management/liferay-portal/assets/67901/41e2e914-cb6b-436c-94d1-175493929e15)

### CODE REVIEW

**Comment by boton** [comment]
ci:test:playwright

**Comment by boton** [comment]
ci:test:relevant

**Comment by boton** [comment]
ci:test:playwright:norebase

File: modules/test/playwright/tests/blogs-web/blogEntry.spec.ts [hunk 1]
```diff
@@ -150,3 +150,57 @@ test('LPD-24858 Categories with blank spaces in friendly URL', async ({
...
+	blogsEditBlogEntryPage,
+	displayPageTemplatesPage,
+	page,
+	pageEditorPage,
+	site,
+}) => {
+	const vocabularyName = getRandomString();
+	const friendlyUrlCategories = ['category-1', 'category-2', 'category-3'];
+
+	const {categories} = await friendlyURLCategoriesSetup({
```
Comment (robertoDiaz) [inline modules/test/playwright/tests/blogs-web/blogEntry.spec.ts]: could we rename it to blogsFriendlyUrlSetup or blogsCategorizedFriendlyUrlSetup

**Review by robertoDiaz — CHANGES_REQUESTED** [review]

File: modules/test/playwright/helpers/HeadlessAdminTaxonomyApiHelper.ts [hunk 2]
```diff
@@ -66,6 +71,20 @@ export class HeadlessAdminTaxonomyApiHelper {
...
 	}
 
+	/**
+	 * It allows update a category name
+	 *
+	 * @param name the new name of the category
+	 * @param id the category id
+	 */
+
+	async patchCategory({id, name}: patchCategoryProps): Promise<{id: number}> {
```
[the remark this answers is not in the document]
Comment (boton) [reply modules/test/playwright/helpers/HeadlessAdminTaxonomyApiHelper.ts:81]: We can go with `postCategory` better than update to apply the existing pattern. 
I can apply the next PR, I'm working on top of this PR.

File: modules/test/playwright/tests/blogs-web/blogEntry.spec.ts (hunk 1 above)
Comment (boton) [reply modules/test/playwright/tests/blogs-web/blogEntry.spec.ts]: We can go with `blogsCategorizedFriendlyUrlSetup`. 
I can apply the next PR, I'm working on top of this PR.

**Comment by boton** [comment]
I applied your comments @robertoDiaz

**Comment by robertoDiaz** [comment]
Good to go! 

👍

**Review by robertoDiaz — APPROVED** [review]

**Pull request closed without merging**